### PR TITLE
Don't review. It will be abandoned. [GPU] Feature/intepolate 3 axes onnx 5d

### PR DESCRIPTION
**Pull request by OlehKravchyshyn**
### Details:
  -  added 3-axis interpolation for linear-onnx mode
  - fixed resample_opt for onnx mode, it didn't work in case of padding
  - added tests for the new implementation and fix

### CODE REVIEW

**Comment by wilson-seok** [comment]
@OlehKravchyshyn Please resolve conflicts to start all build checks.

File: src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/resample_onnx_5d.cl [hunk 1]
```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2018-2022 Intel Corporation
```
Comment (tgubanova-lohika) [inline src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/resample_onnx_5d.cl:1]: Maybe it's better to add "opt" to define that it's specific optimal kernel?
resample_opt_onnx_5d.cl?

**Comment by wilson-seok** [comment]
@OlehKravchyshyn Please check resample unit test failures.
image
They have c++ exception.
image

File: src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/resample_onnx_5d.cl (hunk 1 above)
Comment (jade-cho) [reply src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/resample_onnx_5d.cl:1]: The resample_opt kernel is already saturated. How about moving all 5D related kernels or all linear_onnx kernels, if you separate the files anyway? It is just suggestion.

**Comment by wilson-seok** [comment]
No performance regression in model zoo models.

File: src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/resample_onnx_5d.cl (hunk 1 above)
Comment (OlehKravchyshyn) [reply src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/resample_onnx_5d.cl:1]: It is a reason why we decided to create a separate kernel. Maybe it makes sense to move all onnx stuff in one file, let's wait till the https://github.com/openvinotoolkit/openvino/pull/12977 is merged and see how all of it be together..

Comment (jade-cho) [reply src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/resample_onnx_5d.cl:1]: > It is a reason why we decided to create a separate kernel. Maybe it makes sense to move all onnx stuff in one file, let's wait till the #12977 is merged and see how all of it be together.

https://github.com/openvinotoolkit/openvino/pull/12977 has been merged. How are you going to do this PR?

Comment (OlehKravchyshyn) [reply src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/resample_onnx_5d.cl:1]: onnx will be moved in a separate kernel. we begin working on that.

**Comment by mhnap-lohika** [comment]
@wilson-seok New PR was created instead of this one - https://github.com/openvinotoolkit/openvino/pull/13796

**Pull request closed without merging**